### PR TITLE
Fix error in QR parsing

### PR DESCRIPTION
**Pull request by dagheyman**

### CODE REVIEW

File: qml/AddCredential.qml [hunk 1]
```diff
@@ -206,36 +206,33 @@ DefaultDialog {
...
+        if (type === null || type === 'TOTP') {
             return 0
         }
         if (type === 'HOTP') {
             return 1
         }
-        return false
     }
 
     function getDigitsIndex(digits) {
```
Comment (emlun) [inline qml/AddCredential.qml]: I feel like we should probably `parseInt` here instead...

Comment (dagheyman) [reply qml/AddCredential.qml]: Not following, parse what int?

Comment (emlun) [reply qml/AddCredential.qml]: I mean in `getDigitsIndex`, below this message. :) Instead of matching explicitly on string values, I mean.

Comment (dagheyman) [reply qml/AddCredential.qml]: Sorry, still not following :) After the change we are comparing ints (digits) with ints (index) to preselect the correct option in the dropdown.

Comment (emlun) [reply qml/AddCredential.qml]: Um, never mind me. Apparently I went colorblind for a bit there. :D

**Pull request merged**